src/components/flash/flash.js

```diff
@@ -305,19 +318,19 @@ class Flash extends React.Component {
           key={ title }
           title={ title }
           open={ this.state.dialogs[title] || false }
-          onCancel={ this.toggleDialog.bind(this, title) }
+          onCancel={ this.toggleDialog(title) }
         >
           { desc }
         </Alert>
       );
 
       // create text for item
-      text = (
+      return (
```

Comment:
OK I think there are two ways of achieving the same thing here without redefining the parameter:
```
function(y) {
  var x = '';
  if (y === 137) {
    x = 'yep'
  }
  return x || y;
}
```

or 
```
function(y) {
  if (y === 137) {
    return 'yep'
  }
  return y;
}
```